gem/tests/test_auth.py

```diff
@@ -118,6 +118,25 @@ def test_add_user_role_from_claims(self):
         self.assertEquals(user.groups.all().count(), 1)
         self.assertEquals(user.groups.first().name, 'data_admin')
 
+    def test_display_username_as_alias_from_claims(self):
+        roles = ['data_admin', ]
+        claims = {
+            'roles': roles,
+            'given_name': 'testgivenname',
+            'family_name': 'testfamilyname',
+            'email': 'test@email.com',
+            'sub': 'e2556752-16d0-445a-8850-f190e860dea4'}
+        user = get_user_model().objects.create(
+            username='testuser', password='password')
+
```

Comment:
Please add an assert here to show that the alias is empty/None at this point.
That way the test is 100% explicit about which code blocks it's testing